vars/cloud.groovy

```diff
@@ -44,3 +44,21 @@ void cleanContainersAndImages(String containerEngine = 'podman') {
         sh "${containerEngine} rmi -f ${it} || date"
     }
 }
+
+/*
+* Create CURL for pushing a new image description to quay.io
+* readmeFile = file that is the description for the image (i.e. path/README.md)
+* registryPwd = pwd used when doing a login to quay.io (i.e. podman login -u=${REGISTRY_USER} -p=${REGISTRY_PWD} quay.io)
+* imageId = image on quay.io with group (i.e. kiegroup/kie-server)
```

Comment:
instead of `imageId`, other methods are using the `namespace/repository` vocabulary of Quay